### DISCUSSION TITLE
OK to use screenshots?

### DISCUSSION THREAD

**Discussion by ERYpTION** [streetcomplete/StreetComplete#5434] [General]
I'm working on an article about OSM (for an online magazine) including an introduction to StreetComplete. Would it be alright to use some screenshots from the app @westnordost ? If so, do I need to add a specific license caption by the images?

Thank you.

**Comment by westnordost** [comment]
Sure, you can use them, thank you for asking! Do you care to post the link here after you released it?

I think using screenshots in an article about the topic falls under Fair Use so no explicit attribution is necessary.

In doubt I guess it could also be redistributed by the most restrictive license of any assets contained in the screenshot which would be CC-BY-SA.
